pages/design-tokens/typesetting/font-family.md

```diff
@@ -237,3 +239,41 @@ Your context and coding style determine how you access USWDS family tokens in co
     <div class="grid-col-5">.font-<code>body</code>-<code>2xl</code>;</div>
   </div>
 </div>
+
+## Setting custom fonts
+
+If you want to use fonts outside of the ones mentioned in USWDS Available Fonts, you have a few options. The complexity of importing and setting other fonts depends on how you’re including that font.
+
+### Option 1: Import a font from the web
+
+If you’re importing a new font from an open source font web directory,often you’ll be including JavaScript at the top of your file, and that loads the font and associates it with a display name. To make sure that happens, you’ll need to make a typeface token that uses that display name and associates a fallback font stack with it. Here’s how to make the customizations to your code:
+
+1. Define: Tell USWDS about the font you’re using by defining a new typeface token in your settings. In the code example, we are using the font Lato.
```

Comment:
It would be helpful to give users more information about where to add this code. One solution would be to add a link to the settings configurations instructions here: 
```suggestion
1. Define: Tell USWDS about the font you’re using by defining a new typeface token in your [settings configuration]({{ site.baseurl }}/documentation/settings/#configuring-custom-uswds-settings). In the code example, we are using the font Lato.
```